subsystem/cluster_test.go

```diff
@@ -2428,6 +2428,23 @@ var _ = Describe("cluster install", func() {
 		Context("reset installation", func() {
 			enableReset, _ := strconv.ParseBool(os.Getenv("ENABLE_RESET"))
 
+			checkClusterProgressFields := func(progress *models.ClusterProgressInfo) {
+				Expect(progress).NotTo(BeNil())
+				Expect(progress.FinalizingStagePercentage).To(Equal(int64(0)))
+				Expect(progress.InstallingStagePercentage).To(Equal(int64(0)))
+				Expect(progress.PreparingForInstallationStagePercentage).To(Equal(int64(0)))
+				Expect(progress.TotalPercentage).To(Equal(int64(0)))
+			}
```

Comment:
You can reuse `expectProgressToBe` function in the same file instead of re-implementing the logic

Reply:
Hi.

Is that thread resolved? Do you still keep the redefinition of the function?